src/metaprob/builtin_impl.clj

```diff
@@ -127,9 +127,9 @@
 (defn metaprob-assert [condition complaint & irritants]
   (binding [*out* *err*]
     (doseq [irritant irritants]
-      (if (mutable-trace? irritant)
-        (do (print "Irritant:")
-            (metaprob-pprint irritant)))))
+      (when (mutable-trace? irritant)
```

Comment:
Now I have to learn about `when`.  I assume it's as in  Common Lisp...

Reply:
Yeah, `when` is just `if` without an else?-branch. The two advantages I has over a single-armed `if` are that it arguably communicates intent better and it obviates the need for a `do` if you want to trigger any side-effects in the body.